app/views/hyrax/dashboard/collections/_form.html.erb

```diff
@@ -7,6 +7,9 @@
       <li>
         <a href="#branding" role="tab" data-toggle="tab"><%= t('.tabs.branding') %></a>
       </li>
+      <li>
+        <a href="#relationships" role="tab" data-toggle="tab"><%= t('.tabs.relationships') %></a>
+      </li>
```

Comment:
The relationships tab needs to be wrapped with `<% if @collection.nestable? %>`, such that it doesn't display the tab when `nestable? false`.